router/transformer/transformer.go

```diff
@@ -150,11 +156,52 @@ func (trans *HandleT) Transform(transformType string, transformMessage *types.Tr
 	return destinationJobs
 }
 
+func (trans *HandleT) ResponseTransform(responseData integrations.DeliveryResponseT, destName string) (statusCode int, respBody string) {
+	rawJSON, err := json.Marshal(responseData)
+	if err != nil {
+		panic(err)
+	}
+	var resp *http.Response
+	var respData []byte
+	url := getResponseTransformURL(destName)
```

Comment:
What if the URL is not implemented on transformer. To ensure we have backwards compatibility if the HTTP request returns 404 , can we use the original statusCode and body?

Reply:
Also in cases of 5xx from transformer we should be using the original sendPost response code to ensure we don't deliver the same data twice to the destination

Reply:
Yes